### PR TITLE
Fruit webApp

### PR DESCRIPTION
**Pull request by touredri**
- [ ] Used react redux
- [ ] used react routes
- [ ] Created a new state variable, filterFruit, in the FruitList component to store the value entered in the search bar.
- [ ] Added an onFilterChange callback function to handle changes in the search input and update the filterFruit state.
- [ ] Modified the useEffect hook in the FruitList component to include the filterFruit state as a dependency, triggering a re-render when the filter value changes.
- [ ] Implemented the filtering logic in the fetchFruitElements function. It compares the fruit name with the filterFruit value and only renders the fruits that match the search query.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,192 @@
...
 
-## Learn More
+<!-- LIVE DEMO -->
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+## 🚀 Live Demo <a name="live-demo"></a>
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+- [**Live Demo**](https://dev--superb-taiyaki-212be8.netlify.app/)
+- [**Video Presentation**](https://www.loom.com/share/5576a817a77745728d3db2d78484d70a)
```
Comment (ghost) [inline README.md]: - [ ] Your loom video currently is not opening up. Kindly recheck the link and ensure that it is working

File: README.md [hunk 2]
```diff
@@ -1,70 +1,192 @@
...
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## Acknowledgments
+Big ups to Microverse team .
+Thanks to https://www.fruityvice.com/ for their API
+Thanks to Unsplash: https://help.unsplash.com/en/articles/2511271-guideline-hotlinking-images also for they image API
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (ghost) [inline README.md]: - [ ] Kindly provide appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: src/redux/FruitSlice.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import { getFruit } from './apiCall';
+
+export const initialState = {
+  data: [],
+};
+const Fruit = createSlice({
+  name: 'fruit',
+  // initialState: {
+  //   data: [],
+  // },
```
Comment (ghost) [inline src/redux/FruitSlice.js]: - [ ] Kindly remove commented-out code to keep your code base clean

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @touredri , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ All test are passing


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/FruitSlice.js (hunk 3 above)
Comment (touredri) [reply src/redux/FruitSlice.js]: Sure

File: README.md (hunk 1 above)
Comment (touredri) [reply README.md]: ok solved @frank1738

File: README.md (hunk 2 above)
Comment (touredri) [reply README.md]: Ok @frank1738

**Review by emmiiorji — APPROVED** [review]
Hi @touredri, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Requested changes have been implemented
- Everything looks good

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**